python/xorbits/deploy/oscar/base_config.yml

```diff
@@ -0,0 +1,86 @@
+services:
```

Comment:
As it is only used by k8s now, we can inherit basic config from Mars.

Reply:
Yeah, maybe `"@inherits": '@default'`, and add the changed part to overwrite the original one.

Reply:
Fix